cmd/main.go

```diff
@@ -73,15 +76,16 @@ func main() {
 	prometheus.Unregister(prometheus.NewGoCollector())
 
 	scConfig = &common.SCConfiguration{
-		EventInCh:  make(chan *channel.DataChan, channelBufferSize),
-		EventOutCh: make(chan *channel.DataChan, channelBufferSize),
-		CloseCh:    make(chan struct{}),
-		APIPort:    apiPort,
-		APIPath:    apiPath,
-		PubSubAPI:  v1pubsub.GetAPIInstance(storePath),
-		StorePath:  storePath,
-		AMQPHost:   amqpHost,
-		BaseURL:    nil,
+		EventInCh:   make(chan *channel.DataChan, channelBufferSize),
+		EventOutCh:  make(chan *channel.DataChan, channelBufferSize),
+		CloseCh:     make(chan struct{}),
+		APIPort:     apiPort,
+		APIPath:     apiPath,
+		PubSubAPI:   v1pubsub.GetAPIInstance(storePath),
+		StorePath:   storePath,
+		AMQPHost:    amqpHost,
+		BaseURL:     nil,
+		HwEventPort: hwEventPort,
```

Comment:
HwEventPort should be  in the plugin, the framework should not bring any plugin dependant variables , use environment varibles